jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -68,6 +69,21 @@ def get_invoices_for_disbursement(partner):
         current_app.logger.info(invoices)
         return invoices
 
+    @staticmethod
+    def get_refund_partial_payment_line_items_for_disbursement(partner) -> List[PaymentLineItemModel]:
+        """Return payment line items with partial refunds for disbursement."""
+        payment_line_items: List[PaymentLineItemModel] = db.session.query(PaymentLineItemModel) \
+            .join(InvoiceModel, PaymentLineItemModel.invoice_id == InvoiceModel.id) \
+            .join(RefundsPartialModel, PaymentLineItemModel.id == RefundsPartialModel.payment_line_item_id) \
+            .filter(InvoiceModel.invoice_status_code == InvoiceStatus.PAID.value) \
+            .filter(InvoiceModel.payment_method_code.in_([PaymentMethod.CC.value, PaymentMethod.DIRECT_PAY.value])) \
```

Comment:
CC is for NSF only, we don't offer partial refunds on NSF

Reply:
removed